src/modal/ovh-angular-sso-auth-modal-plugin.html

```diff
@@ -1,11 +1,16 @@
 <div id="sso-modal" class="oui-modal oui-modal_shadow">
-    <div class="oui-modal__header" data-ng-if="!ssoAuthModalCtrl.loaders.trads">
-        <button class="oui-icon oui-icon-close_thin oui-modal__close-button" type="button" aria-label="Exit"></button>
-    </div>
+    <div class="oui-modal__header"></div>
 
-    <div class="oui-modal__body oui-modal__body_no-icon" data-ng-if="ssoAuthModalCtrl.loaders.init" data-translate="sso_modal_loading"></div>
+    <div class="oui-modal__body oui-modal__body_no-icon text-center" data-ng-if="ssoAuthModalCtrl.loaders.init">
+        <div class="oui-loader oui-loader_s">
+            <div class="oui-loader__container m-auto center-block">
+                <div class="oui-loader__image"></div>
+            </div>
+        </div>
+        <span data-translate="sso_modal_loading"></span>
```

Comment:
this translation is not _necessarily_ resolved as the translation might still be loading => add a `data-ng-if="!ssoAuthModalCtrl.loaders.trads"` ?